### PR TITLE
DHFPROD-5412: Can't clear/unselect selected facets on sidebar

### PR DESCRIPTION
**Pull request by saarimrahman**
### Description
- includes a fix for DHFPROD-5408: Discard all changes icon in explorer screen not works consistently
- hub properties now persist after selecting a hub property
- removed entity name from showing in Collection facet under hub properties

#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [ ] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub-central/ui/src/components/selected-facets/selected-facets.tsx [hunk 1]
```diff
@@ -139,7 +138,7 @@ const SelectedFacets: React.FC<Props> = (props) => {
                 data-cy={`clear-${item.rangeValues.lowerBound}`}
               >
                 <Icon type='close'/>
-                {facetName + ': ' + item.rangeValues.lowerBound + ' ~ ' + item.rangeValues.upperBound}
```
Comment (Sanjeevani19) [inline marklogic-data-hub-central/ui/src/components/selected-facets/selected-facets.tsx:142]: @saarimrahman any reason why facetName is removed ? As per the requirement it should be facetName:values right ?

Comment (saarimrahman) [reply marklogic-data-hub-central/ui/src/components/selected-facets/selected-facets.tsx:142]: See Yash's comment here: https://project.marklogic.com/jira/browse/DHFPROD-5412?focusedCommentId=181039&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-181039

Comment (Sanjeevani19) [reply marklogic-data-hub-central/ui/src/components/selected-facets/selected-facets.tsx:142]: @saarimrahman I think you misunderstood the requirement, Yash comment was to not show the facetName which is same as entityName

**Comment by timur-isangulov** [comment]
It shows apply all changes/discard all changes icons when I select previously saved query but should not. 
Screen Shot 2020-08-11 at 3 20 16 PM

**Comment by akshaysonvane** [comment]
Closing this as @Sanjeevani19 is going to create a separate PR for this task.

**Pull request closed without merging**